script/run-all.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
-
-# This script is useful for regenerating all of the smoke tests running locally.
-
-declare -a arr=("actions" "bundler" "cargo" "composer" "docker" "elm" "go" "gradle" "hex" "maven" "npm" "nuget" "pip" "pip-compile" "pipenv" "poetry" "pub" "submodules" "terraform")
-for eco in "${arr[@]}"
-do
-  go run cmd/dependabot/dependabot.go test -f "testdata/smoke-$eco.yaml" -o "testdata/smoke-$eco.yaml"
```

Comment:
These `testdata/smoke-*` files don't exist in this repo anymore, they now live here: https://github.com/dependabot/smoke-tests/tree/cd5c5b5d8fb9abbd0f6db98c347dd9f7d065c6d4/tests